packages/twenty-server/src/engine/core-modules/auth/auth.resolver.ts

```diff
@@ -83,6 +84,7 @@ export class AuthResolver {
     });
   }
 
+  @UseGuards(CaptchaGuard)
```

Comment:
Let's also add it to checkUserExists to avoid brutforce-guessing user emails?

Reply:
Done